src/components/layout/footer.js

```diff
@@ -1,216 +1,197 @@
     font-weight: bold;
     color: var(--color-black-3);
-    font-size: var(--text-size-s);
+    font-size: var(--text-size-xs);
+    text-decoration: none;
+
+    :hover {
+        text-decoration: underline;
+    }
     @media ${device.tabletL} {
         font-size: var(--text-size-sm);
     }
 `
-const BoldLink = styled(StyledLink)`
-    ${BoldSharedCss}
-`
-const Risk = styled(Text)`
-    ${BoldSharedCss}
-    margin-left: 0.9rem;
+const RiskWarning = styled.div`
+    background-color: var(--color-grey-26);
+    border-left: 4px solid #f8a2a8;
+    padding: 1.6rem;
+    margin-top: 2rem;
+
+    @media ${device.tabletL} {
+        border-top: 2px solid #f8a2a8;
+        border-left: none;
+        width: 90%;
+        margin: 4rem auto 0;
+        padding: 1rem;
+
+        p {
+            padding: 0;
+            margin: 0;
+            width: 100%;
+        }
+    }
 `
-const StaticAsset = styled.a`
+
+const BoldLink = styled(StyledLink)`
     font-weight: bold;
     color: var(--color-black-3);
-    font-size: var(--text-size-s);
-    text-decoration: none;
+    font-size: var(--text-size-xs);
+    @media ${device.tabletL} {
+        font-size: var(--text-size-sm);
+    }
+`
+const Copyright = styled(Flex)`
+    grid-area: copyright;
+    justify-content: flex-start;
+    align-items: center;
 
-    :hover {
-        text-decoration: underline;
+    p {
+        font-size: var(--text-size-xs);
+        line-height: 1.14;
     }
+
     @media ${device.tabletL} {
-        font-size: var(--text-size-sm);
+        width: 90%;
+        margin: 0 auto;
+        padding: 2rem 0;
+
+        p {
+            font-size: 1.75rem;
+            line-height: 1.5;
+        }
     }
 `
-/* TODO: [social-media] uncomment when social media accounts are ready for Deriv */
-// const ExternalLink = styled.a`
-//     text-decoration: none;
-// `
+const SocialWrapper = styled.div`
+    grid-area: social;
+    margin: 1.6rem 0;
 
-// const SocialWrapper = styled(Flex)`
-//     svg {
-//         width: 4.2rem;
-//         margin-right: 1rem;
-//     }
-//     ${Text} {
-//         margin-top: 0;
-//         letter-spacing: 2px;
-//         color: var(--color-black-6);
-//         margin-bottom: 0.8rem;
+    svg {
+        margin-left: 1.6rem;
+    }
 
-//         @media ${device.tabletL} {
-//             margin-bottom: 1rem;
-//         }
-//     }
-// `
-const SocialMedia = styled(Flex)`
     @media ${device.tabletL} {
-        margin-top: 2rem;
-        flex-direction: row;
+        display: flex;
+        justify-content: center;
+
+        a:first-child {
+            svg {
+                margin-left: 0;
+            }
+        }
     }
 `
+const ExternalLink = styled.a`
+    text-decoration: none;
+`
 const MobileAccordion = styled.section`
-    border-top: 1px solid var(--color-red);
-    background-color: var(--color-grey-8);
+    background-color: var(--color-grey-25);
 
     p {
-        letter-spacing: 2px;
+        font-size: 2rem !important;
```

Comment:
why?, i think instead of pointing style to `p`, we can use another styled component for this